### DISCUSSION TITLE
Alarm setting: how to display a value within a Pop up message

### DISCUSSION THREAD

**Discussion by mansonchan888** [artisan-roaster-scope/artisan#1134] [Q&A]
I would like to show the current BT (Y2) during a message  popup or  "/usr/bin/say  a current temperature"
can I do that? and how?
<img width="759" alt="WeChatc5c3f2d5b63369d2553d3e46b6c442b0" src="https://user-images.githubusercontent.com/119805088/226489471-b4387617-f82b-4fa8-ab4f-46709ba17e94.png">

**Comment by bigl00z3** [comment]
From my understanding I don't think you can do it (and the lack of answers are probably going in the same direction).

Here you can only enter a single command and the attribute `{BT}` (which is mentioned in the Help page for Events) would not work in this case.
Others feel free to correct me if I'm wrong.
Nico
